### ISSUE TITLE
Minor comments

### ISSUE THREAD

**Issue by Bultako** [gammapy/icrc2017-gammapy-poster#10]
Poster 
- Abstract and Getting started blocks: "For further information, visit docs.gammapy.org and follow the links to 'tutorials' and "CTA'".  I do not find the link "CTA" in http://docs.gammapy.org
- In general, keep in the poster the final version of the text written in the proceedings. (i.e. suggestions from Antonio Stamerra and Lars)

Proceedings
- Introduction: "...southern (Chile) an one in the northern (La Palma)..." ->  "...southern (Chile) an one in the northern (La Palma, Spain)..."
- Gammapy package: Second paragraph "An example script [...] www.iausofa.org." is specially hard to follow
- Gammapy package: "Gammapy aims to be a base package [...] can build."-> "Gammapy aims to be a base package for VHE analysis. More specialized packages such as Fermipy (github.com/fermipy/fermipy) for Fermi-LAT data analysis or Naima [15] for the modeling of non-thermal spectral energy distributions of astrophysical sources could leverage and built on Gammapy functionalities."

**Comment by cdeil** [comment]
@Bultako - Thank you very much!

I think all of your comments should be taken into account in the updated poster and proceeding?

https://github.com/gammapy/icrc2017-gammapy-poster/blob/master/proceeding/gammapy-icrc2017-v3.pdf
https://github.com/gammapy/icrc2017-gammapy-poster/blob/master/poster/icrc2017-gammapy-poster-v3.pdf

For the poster, we'll print it tomorrow and it's final.

For the proceeding, we can submit another final version after ICRC (deadline end of July), so for that comments / suggestions are still welcome.

I'm closing this issue now.
